src/test/regress/expected/shard_rebalancer.out

```diff
@@ -2378,3 +2378,44 @@ SELECT rebalance_table_shards();
 
 CALL citus_cleanup_orphaned_shards();
 DROP TABLE test_rebalance_with_index CASCADE;
+-- Test rebalancer with disabled worker
+SET citus.shard_replication_factor TO 2;
+DROP TABLE IF EXISTS test_rebalance_with_disabled_worker;
+CREATE TABLE test_rebalance_with_disabled_worker (a int);
+SELECT create_distributed_table('test_rebalance_with_disabled_worker', 'a');
```

Comment:
minor: maybe add colocate_with:='none' such that other tests do not interfere with this test